firefox-ios/Client/Frontend/Library/Reader/ReaderPanelEmptyStateView.swift

```diff
@@ -0,0 +1,166 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import UIKit
+import Common
+
+private struct ReadingListPanelUX {
```

Comment:
nit: what do you think about adding this within the `UIView` class and follow the same convention in naming:
`private struct UX {` similar to other areas of the application?

Reply:
Great suggestion! Thank you for pointing that out. Please continue to flag these things for me!